{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests22_6_1.py::test_sum_is_divisable_by_n_2", "location": ["tests22_6_1.py", 14, "test_sum_is_divisable_by_n_2"], "keywords": {"test_sum_is_divisable_by_n_2": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00016740101273171604, "start": 1729724384.20456, "stop": 1729724384.2047281, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw2>"}
{"nodeid": "tests22_6_1.py::test_one", "location": ["tests22_6_1.py", 3, "test_one"], "keywords": {"test_one": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0002665890206117183, "start": 1729724384.2045655, "stop": 1729724384.2048333, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw0>"}
{"nodeid": "tests22_6_1.py::test_sum_of_evens_is_even", "location": ["tests22_6_1.py", 24, "test_sum_of_evens_is_even"], "keywords": {"test_sum_of_evens_is_even": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0002038870006799698, "start": 1729724384.2044775, "stop": 1729724384.2046816, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw1>"}
{"nodeid": "tests22_6_1.py::test_sum_is_divisable_by_n_2", "location": ["tests22_6_1.py", 14, "test_sum_is_divisable_by_n_2"], "keywords": {"test_sum_is_divisable_by_n_2": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_sum_is_divisable_by_n_2():", "        n = random.randint(1, 1000)", ">       assert find_sum_of_multiples(n) / n == (474 * (474 + 1)) / 2"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 17, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(23161842285, 281)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_sum_is_divisable_by_n_2():", "        n = random.randint(1, 1000)", ">       assert find_sum_of_multiples(n) / n == (474 * (474 + 1)) / 2"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 17, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(23161842285, 281)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (126678430910016) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.9995932550227735, "start": 1729724384.2050006, "stop": 1729724391.2045946, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw2>"}
{"nodeid": "tests22_6_1.py::test_sum_is_divisable_by_n_2", "location": ["tests22_6_1.py", 14, "test_sum_is_divisable_by_n_2"], "keywords": {"test_sum_is_divisable_by_n_2": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (126678430910016) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00012461599544622004, "start": 1729724391.215288, "stop": 1729724391.215413, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw2>"}
{"nodeid": "tests22_6_1.py::test_one", "location": ["tests22_6_1.py", 3, "test_one"], "keywords": {"test_one": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_one():", "        given_number = 1", "        expected_result = (474 * (474 + 1)) / 2", ">       assert find_sum_of_multiples(given_number) == expected_result"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(188704808)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_one():", "        given_number = 1", "        expected_result = (474 * (474 + 1)) / 2", ">       assert find_sum_of_multiples(given_number) == expected_result"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(188704808)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139203857876544) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.999469802976819, "start": 1729724384.205191, "stop": 1729724391.2046618, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw0>"}
{"nodeid": "tests22_6_1.py::test_one", "location": ["tests22_6_1.py", 3, "test_one"], "keywords": {"test_one": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139203857876544) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.0001353150000795722, "start": 1729724391.220901, "stop": 1729724391.2210371, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw0>"}
{"nodeid": "tests22_6_1.py::test_sum_of_evens_is_even", "location": ["tests22_6_1.py", 24, "test_sum_of_evens_is_even"], "keywords": {"test_sum_of_evens_is_even": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_sum_of_evens_is_even():", "        n = random.randint(1, 1000)", ">       assert find_sum_of_multiples(n) % 2 == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 27, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(152574345)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_sum_of_evens_is_even():", "        n = random.randint(1, 1000)", ">       assert find_sum_of_multiples(n) % 2 == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/tests22_6_1.py", "lineno": 27, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return sum((n * multiples for n in count(multiples, multiples)))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "in find_sum_of_multiples"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   return sum((n * multiples for n in count(multiples, multiples)))", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "count(152574345)"]]}, "reprlocals": null, "reprfileloc": {"path": "Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q22/codegemma_results_1/Folder_6/generated_answer.py", "lineno": 4, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (127587560982080) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.999794074014062, "start": 1729724384.20497, "stop": 1729724391.2047648, "extras": [], "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw1>"}
{"nodeid": "tests22_6_1.py::test_sum_of_evens_is_even", "location": ["tests22_6_1.py", 24, "test_sum_of_evens_is_even"], "keywords": {"test_sum_of_evens_is_even": 1, "tests22_6_1.py": 1, "Folder_6": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (127587560982080) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00016798899741843343, "start": 1729724391.2279356, "stop": 1729724391.228104, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "87b39ec71db541e49f14c9d7a764fbe7", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
